extensions/mssql/test/unit/tableExplorerWebViewController.test.ts

```diff
@@ -384,8 +384,12 @@ suite("TableExplorerWebViewController - Reducers", () => {
                     LocConstants.TableExplorer.rowMarkedForRemoval,
                 ),
             ).to.be.true;
-            expect(controller.state.resultSet?.rowCount).to.equal(1);
-            expect(controller.state.resultSet?.subset.length).to.equal(1);
+            // Row should still be in resultSet (not physically removed, just marked for deletion)
+            expect(controller.state.resultSet?.rowCount).to.equal(2);
+            expect(controller.state.resultSet?.subset.length).to.equal(2);
```

Comment:
.to.have.length(2) woudl result in a more useful error message.  Can you convert the others while you're here?